### PR TITLE
Show missing options when confirming attendance and recording absence

### PR DESCRIPTION
**Pull request by benilovj**
## Trello

https://trello.com/c/eEhlOdMY

## Service user complied

image

## Service user didn't comply – behavioural issues

image

## Service user didn't comply – other reason (unacceptable behaviour)
image

## Service user absent – unacceptable absence

image

## Service user absent – acceptable absence

image

### CODE REVIEW

**Comment by benilovj** [comment]
@duffylouise I see what you mean – those bits are structured data taken directly from Delius, where that's the case being used. I'd suggest we leave it as is for now, and look at this again as part of the work that @leighchristie-gov is doing around the structured data around non-compliance data...

File: app/views/confirm-attendance/_outcome-details.html [hunk 1]
```diff
@@ -22,6 +30,32 @@
...
     },
+    {
+      key: { text: "Reason for not complying" },
+      value: { text: entry['non-compliance-reason'] if entry['non-compliance-reason'] !== 'Other' else entry['non-compliance-reason-other'] },
+      actions: {
+        items: [
+          {
+            href: "compliance",
+            text: "Change",
+            visuallyHiddenText: "complied"
```
Comment (fofr) [inline app/views/confirm-attendance/_outcome-details.html:41]:
```suggestion
            visuallyHiddenText: "reason for not complying"
```

File: app/views/confirm-attendance/check.html [hunk 2]
```diff
@@ -6,6 +6,7 @@
   <h1 class="govuk-heading-xl">{{ title }}</h1>
 
   {% set showAction = true %}
+  {% set entry = data['communication'][CRN][sessionId] %}
```
Comment (fofr) [inline app/views/confirm-attendance/check.html:9]: I wonder if this sort of thing should be set as a local var in routes.

**Review by fofr — APPROVED** [review]

File: app/views/confirm-attendance/check.html (hunk 2 above)
Comment (benilovj) [reply app/views/confirm-attendance/check.html:9]: Yes, agreed – will make this change on a subsequent PR.

**Pull request merged**